### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by longrunningprocess** [saadeghi/daisyui#585] [General]
Just wanted to thank you, your work is exceptional and I'm benefiting from it greatly.  I hope you are getting all you deserve for your outstanding understanding of ux/dx... you're walking an amazing line and I'm grateful I'm came across your stuff, I've been inspired 🚀

**Comment by longrunningprocess** [comment]
I wasn't able to DM you on Twitter but wanted to make sure you got the message.

**Comment by saadeghi** [comment]
Very kind of you 💙
